service_capacity_modeling/models/org/netflix/crdb.py

```diff
@@ -138,10 +138,10 @@ def _estimate_cockroachdb_cluster_zonal(
     max_local_disk_gib: int = 2048,
     max_regional_size: int = 288,
     max_rps_to_disk: int = 500,
+    min_vcpu_per_instance: int = 4,
```

Comment:
minor nit: I'd just call it `cpu` since vcpu isn't really a thing outside AWS

Reply:
https://www.cockroachlabs.com/docs/stable/common-issues-to-monitor.html#cpu-planning CRDB uses this term broadly, more like a cloud thing?